readme.md

```diff
@@ -197,6 +214,13 @@ Default: `false`
 
 Suppress output produced by the PHP-server.
 
+### settings
```

Comment:
You're right. Sould be concurrent to the PHP naming. Renamed it to `directives`.